server/db/cassandra/dataGenerator.js

```diff
@@ -0,0 +1,80 @@
+const faker = require('faker');
+const log = require('fancy-log');
+const fs = require('fs');
+const path = require('path');
+
+
+function createTickers() {
+    const alphabet = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z'];
```

Comment:
Why not use faker random alphanumeric?

Reply:
With this many records, I would end up with duplicates.  Better to systematically generate all possible combinations of 4-letter ticker symbols and 5-letter ticker symbols.